less/navigation.less

```diff
@@ -1 +1,45 @@
-// Navigation code here
\ No newline at end of file
+// Navigation code here
+
+@keyframes space {
+    0% {color: @header-font-color;}
+    25% {color: navy;}
+    50% {color: darkmagenta;}
+    75% {color: purple;}
+    100% {color: lightskyblue;}
+}
+
+.nav-bar {
+    width: 100%;
+    height: 100px;
+    border-bottom: 2px solid rgba(241,241,241,0.8);
+    background-color: rgba(246,246,246,0.8);
+    padding: 10px 0;
+    position: sticky;
+    top: 0;
+
+    .flexinColumn();
+
+    h1 {
+        margin-bottom: 15px;
+        animation-name: space;
+        animation-duration: 10s;
+        animation-delay: 2s;
+        animation-iteration-count: 3;
+    }
```

Comment:
I really love your animation of the SpaceWalkers header.  Really well done! @desiquinn